### ISSUE TITLE
Unrecognized protocol is blamed on currently viewed site

### ISSUE THREAD

**Issue by webdesserts** [beakerbrowser/beaker#999]
Operation System: Mac OS 10.13.4
Beaker version: f88eef1

If I try to navigate to an unrecognized protocol, Beaker will complain about the currently viewed site trying to access that url, even though it was the user who requested it.

![i-never-trusted-this-site-anyway](https://user-images.githubusercontent.com/916253/40396945-76480b3a-5df5-11e8-8dd9-b2687e833782.gif)

**Comment by pfrazee** [comment]
Yeah the reason for that is, traditionally the O.S. handles nonstandard protocols. A really good (and really obnoxious) example is the itunes:// URL, which launches itunes' store. To make sure stupid apps aren't getting launched on you all the time, browsers ask the user for permission... but it doesnt really work right when you're manually entering the URL, does it?

I'm sort of inclined to just have nonstandard protocols fail.
